cmake/test/gtest.cmake

```diff
@@ -1,4 +1,5 @@
 _generate_function_if_testing_is_disabled("catkin_add_gtest")
+_generate_function_if_testing_is_disabled("catkin_add_gmock")
```

Comment:
This doesn't work since the first invocation already calls `return`.

I addressed this in https://github.com/ros/catkin/pull/897/commits/aa28d0c8acdb48846cd53945f9e05b7d8ec56fe5.